atomic_reactor/util.py

```diff
@@ -953,3 +953,24 @@ class LabelFormatter(string.Formatter):
     """
     def get_field(self, field_name, args, kwargs):
         return (self.get_value(field_name, args, kwargs), field_name)
+
+
+def do_retries(method, url, client_statuses=HTTP_CLIENT_STATUS_RETRY,
```

Comment:
`raise_for_status` is provided as a method on the Response class, so you can use it as above even when using a session.